kolibri/utils/main.py

```diff
@@ -160,6 +161,28 @@ def _setup_django():
         raise
 
 
+def _copy_preseeded_db(db_name, target=None):
+    target = target or "{}.sqlite3".format(db_name)
+    target = os.path.join(KOLIBRI_HOME, target)
+    if not os.path.exists(target):
+        try:
+            import kolibri.dist
+
+            db_path = os.path.join(
+                kolibri.dist.__file__, "home/{}.sqlite3".format(db_name)
```

Comment:
Using `__file__ `here is adding `__init__.py` to the path creating something like `.. kolibri/kolibri/dist/__init__.py/home/db.sqlite3 ` .
To make it work I had to change it to 
`os.path.join(kolibri.dist.__path__[0], "home/{}.sqlite3".format(db_name))`

Reply:
Wrapped in `os.path.dirname`